src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -175,6 +184,24 @@ public void ClipToContent(ShadowPaintState state, SKCanvas canvas)
 			canvas.ClipRoundRect(GetContentShape(state), antialias: true);
 		}
 
+		public static bool IsCornerRadiusProportional(CornerRadius CornerRadius)
```

Comment:
rename parameter to `CornerRadius cornerRadius`

Reply:
Done.